doc/docs/usage/tutorial-vehicle-signals.md

```diff
@@ -340,141 +340,3 @@ dependencies:
 ```
 
 The next time the Ankaios server and the two agents will be started, this startup config will be applied.
-
-## Define re-usable configuration
```

Comment:
If the vehicle signals tutorial does not contain workload configs in action anymore, then the sentence link in the startup configuration chapter must be adapted as well: https://eclipse-ankaios.github.io/ankaios/main/reference/startup-configuration/#configuration-structure

Reply:
Done